### ISSUE TITLE
Support loading certificates installed locally

### ISSUE THREAD

**Issue by ulyssa** [ulyssa/iamb#84]
I've been using the `rustls` crate in iamb, but it doesn't currently load locally installed certificates. I can use the `rustls_native_certs` crate to take care of that.

Additionally, I should probably add a feature to support compiling with OpenSSL instead, since I assume it probably handles locally installed certs in a way people are more used to.

**Comment by 0323pin** [comment]
> Additionally, I should probably add a feature to support compiling with OpenSSL instead

Would you mind pinging me if you do this? It's not an issue  but, it requires changes to our package, as I will need to link base openssl at compile time.

TIA

**Comment by ulyssa** [comment]
@0323pin I've added support for building linking against the system libraries. I've put some instruction in a [PACKAGING.md](https://github.com/ulyssa/iamb/blob/main/PACKAGING.md) file. Let me know if you have any questions!

**Comment by 0323pin** [comment]
@ulyssa Thank you for the heads-up.
Since the default features didn't change, I don't think I'll do any changes to our package. I try to keep defaults as far as possible for my packages.

I'll start a build from git-HEAD later today and will let you know if there're issues

**Comment by 0323pin** [comment]
@ulyssa I think you forgot to update `Cargo.lock`

```
===> Building for iamb-0.0.9
error: no matching package found
searched package name: `hyper-tls`
perhaps you meant:      hyper-rustls
location searched: registry `crates-io`
required by package `reqwest v0.11.22`
    ... which satisfies dependency `reqwest = "^0.11.10"` (locked to 0.11.22) of package `matrix-sdk v0.7.1`
    ... which satisfies dependency `matrix-sdk = "^0.7.1"` (locked to 0.7.1) of package `iamb v0.0.9-alpha.1 (/usr/pkgsrc/wip/iamb-dev/work/iamb-9c6ff58b968f1b5bcfe995b8d65761160b98c3e9)`
As a reminder, you're using offline mode (--offline) which can sometimes cause surprising resolution failures, if this error is too confusing you may wish to retry without the offline flag.
*** Error code 101

Stop.
```

**Comment by ulyssa** [comment]
Thank you for catching that! I've fixed it in #219.

**Comment by 0323pin** [comment]
@ulyssa All good with both features. I've, both built and tested the resulting binaries :+1:
